african-market-place/src/components/pages/Login/LoginContainer.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import RenderLoginPage from './RenderLoginPage';
+
+const LoginContainer = () => {
+  return (
+    <>
+      <RenderLoginPage />
+    </>
+  );
+};
+
+export default LoginContainer;
```

Comment:
I don't think this file is necessarily needed. We can just call LoginPage in app.js when we want it to render